env_kind.go

```diff
@@ -767,20 +766,27 @@ func (r *kindRunnable) prePullImage(ctx context.Context) (err error) {
 		return errors.Newf("service %q is running; expected stopped", r.Name())
 	}
 
-	if _, err = r.env.execContext(ctx, "docker", "image", "inspect", r.opts.Image).CombinedOutput(); err == nil {
-		return nil
+	if image, err := r.env.execContext(ctx, "docker", "image", "ls", r.opts.Image, "--format", "{{.Repository}}:{{.Tag}}").CombinedOutput(); err != nil {
```

Comment:
Not sure I understand the benefit of not using docker image inspect here? 
If the image isn't present locally it would respond with No such image, and you can pull?

Reply:
done